tensorflow/python/kernel_tests/cholesky_grad_op_test.py

```diff
@@ -0,0 +1,154 @@
+import tensorflow as tf
+
+import numpy as np
+import scipy.linalg as linalg
+
+from scipy.linalg.blas import dsymv
+
+#Reference Python implementation ported from Iain Murray Matlab code. Blocked version.
```

Comment:
While these tests were probably very helpful for developing the code, they are not needed as unit tests when checking this into TensorFlow. The standard approach is to have a test that compares the analytical gradients against a finite difference approximation. This functionality is provided by the tf.test.compute_gradient_error() method. You can find plenty of examples in tensorflow/python/ops/*_grad_test.py.